std_msgs/Header header
bool is_detected
sensor_msgs/CompressedImage rgb
sensor_msgs/CompressedImage depth
BBox[] bbox
sensor_msgs/CompressedImage[] segments
Pose3D[] pose
